hello-thirdparty/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.9-all.zip
```

Comment:
As you want, but gradle-experimental:0.6.0+ requires gradle 2.9, so you can do that in a separate change only if you do it before migrating any sample to gradle-experimental:0.6.0+.